lib/cs_guide/resources/venue.ex

```diff
@@ -9,8 +9,14 @@ defmodule CsGuide.Resources.Venue do
 
   schema "venues" do
     field(:venue_name, :string)
+    field(:description, :string)
+    field(:address1, :string)
```

Comment:
`:address` is discussed in: https://github.com/dwyl/fields/issues/21
we _must_ create and use the _modular/reusable_ field 
**so that** we don't have to keep re-writing address validation and sanitisation code `forEach` new project.